CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ Release date: TBD
 
 ### Bug Fixes
 
+#### All platforms
+- An existing application instance will be reused instead of starting another instance.
```

Comment:
My boss said, it's a bug :)
<img width="218" alt="napkin 17 08 16 10 38 41 am" src="https://cloud.githubusercontent.com/assets/4549976/17729951/dc596444-6466-11e6-93fb-d0f63385816d.png">